automated-release/index.js

```diff
@@ -1,74 +1,78 @@
 const Github = require('github-release')
-const Artifacts = require('artifact-collect')
+const collector = require('artifact-collect')
 const GitBranch = require('git-branch')
-const fs = require('fs-extra')
+const AndroidProps = require('android-release-properties')
```

Comment:
Discussed offline: it should *not*  have "./" as it would change the meaning from deployed modules to local files. 

Worth noticing how local packages are declared in the `package.json` file could be different from Linux to MacOS